backend/src/v1/allegation_complaint/allegation_complaint.service.ts

```diff
@@ -545,12 +545,18 @@ export class AllegationComplaintService {
     }
 
     if (officerAssigned) {
-      builder.andWhere("people.person_complaint_xref_code = :Assignee", {
-        Assignee: "ASSIGNEE",
-      });
-      builder.andWhere("people.person_guid = :PersonGuid", {
-        PersonGuid: officerAssigned,
-      });
+      if (officerAssigned === "Unassigned")
+      {
+        //Special case for unassigned 
+        builder.andWhere("people.person_complaint_xref_guid IS NULL");
+      } else {
+        builder.andWhere("people.person_complaint_xref_code = :Assignee", {
+          Assignee: "ASSIGNEE",
+        });
+        builder.andWhere("people.person_guid = :PersonGuid", {
+          PersonGuid: officerAssigned,
+        });
+      }
     } else if (officerAssigned === "null") {
```

Comment:
This part seems like a bug, but it's not part of your changes.  Officer assigned is being compared to the string "null", not to null.

Reply:
It doesn't seem to do anything so I nuked it.